xml/System.Text.Json/Utf8JsonReader.xml

```diff
@@ -203,9 +264,22 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Reads the next JSON token value from the source and parses it to a <see cref="T:System.Double" />.</summary>
+        <returns>The UTF-8 encoded token value parsed to a <see cref="T:System.Double" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+
+## Remarks
+
+The `GetDouble` method throws an exception if the entire UTF-8 encoded token value cannnot be parsed to a <xref:System.Double> value.
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.InvalidOperationException">
+            The JSON token value isn't a <see cref="F:System.Text.Json.JsonTokenType.Number" />.</exception>
```

Comment:
Should all of these InvalidOperationException messages use the more-Englishy `The JSON token value isn't <see cref="Number"/>` or the more programmery `<see cref="TokenType" /> is not <see cref="Number" />`?

Reply:
It renders like this:
![image](https://user-images.githubusercontent.com/12971179/55599245-a241f880-570b-11e9-9204-984e62696577.png)

i think it's fine as-is?